### PR TITLE
Updates README with new ILP gRPC URL

### PR DESCRIPTION
**Pull request by amiecorso**
## High Level Overview of Change
Updates the README to contain correct ILP gRPC URL.

### Type of Change

- [x] Documentation Updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -324,7 +324,7 @@ All calls to `IlpClient` must pass an access token, which can be generated in yo
 ```java
 import io.xpring.ilp.IlpClient;
 
-String grpcUrl = "hermes-envoy-test.xpring.io"; // Testnet Hermes URL
+String grpcUrl = "prod.grpcng.wallet.xpring.io:443"; // Testnet ILP Wallet URL
```
Comment (sappenin) [inline README.md:327]: Does the client need the `443` port, or can that be inferred?

Comment (keefertaylor) [reply README.md:327]: I don't think it can be inferred because this isn't http traffic (though I could be wrong?)

Comment (nkramer44) [reply README.md:327]: Tested this locally.  Including the port actually doesn't work, so we should get rid of the port here as well

Comment (amiecorso) [reply README.md]: removed

**Pull request merged**